### DISCUSSION TITLE
How to properly use override password

### DISCUSSION THREAD

**Discussion by peterle-qwerty** [irrdnet/irrd#642] [General]

**Comment by troy2914** [comment]
It needs to be a seeded salt `$1$seed$` type crypt.    looks like you created it with a "md5sum" type command.

**Comment by dhutchin-ctl** [comment]
We figured that out.  Our initial problem was the hasher I was using put out hex encoding instead of base64 or whatever the printable ascii encoding is.  We has already tried the seed, and once we used a different hasher things worked fine.  We we confused because we thought it had worked a week ago, but that was on 4.2.2.  Don't know if it had a security problem. Anyway, thanks for the quick response.

**Comment by mxsasha** [comment]
FYI @peterle-qwerty, the password you included in your now-deleted example is still available to others in the discussion history, and will have been mailed to a number of people, so consider that unsafe. But I'm assuming that was an example and not a password you actually use somewhere ;)
